src/app/results/results.component.css

```diff
@@ -16,7 +16,6 @@
   text-overflow: ellipsis;
 }
 .infobox {
-  float:right!important;
 
 }
```

Comment:
I think it's as well that you remove it, people can add the styling for this class as needed 👍